src/Workspaces/Core/Portable/ExtensionManager/IExtensionManager.cs

```diff
@@ -5,14 +5,15 @@
 using System;
 using Microsoft.CodeAnalysis.Host;
 
-namespace Microsoft.CodeAnalysis.Extensions
-{
-    internal interface IExtensionManager : IWorkspaceService
-    {
-        bool IsDisabled(object provider);
+namespace Microsoft.CodeAnalysis.Extensions;
 
-        bool CanHandleException(object provider, Exception exception);
+internal interface IExtensionManager : IWorkspaceService
+{
+    bool IsDisabled(object provider);
 
-        void HandleException(object provider, Exception exception);
```

Comment:
merged these into one helper.  much simpler and cleaner.  especially as some CanHandleException helpers were stateful!